config/chainspec.toml

```diff
@@ -96,129 +106,205 @@ compute_rewards = true
 # Defines how refunds of the unused portion of payment amounts are calculated and handled.
 #
 # Valid options are:
-#   'refund': this causes excess payment amounts to be sent to either a pre-defined purse, or back to the sender.
-#             the refunded amount is calculated as the given ratio of the payment amount minus the execution costs.
-#   'burn': similar to what refund does; except the refund amount is burned.
+#   'refund': a ratio of the unspent token is returned to the spender.
+#   'burn': a ratio of the unspent token is burned.
+#   'no_refund': no refunds are paid out; this is functionally equivalent to refund with 0% ratio.
+# This causes excess payment amounts to be sent to either a
+# pre-defined purse, or back to the sender.  The refunded amount is calculated as the given ratio of the payment amount
+# minus the execution costs.
 refund_handling = { type = 'refund', refund_ratio = [99, 100] }
```

Comment:
Didn't we change this to 75 percent or something?

Reply:
I see 75 in my notes, but missed pulling it forward. Corrected.